esinstall/src/index.ts

```diff
@@ -359,7 +359,7 @@ export async function install(
         cwd,
         packageLookupFields,
       });
-      if (resolvedResult.type === 'JS') {
+      if (resolvedResult.type === 'BUNDLE') {
```

Comment:
Do we not need `JS` handling anymore?

Reply:
yea, renamed to something more inclusive of files like `.svelte`. Basically: should this be bundled by Rollup, or should this be copied over 1:1 as an asset and not passed through rollup.